src/state/page-config/actions.js

```diff
@@ -265,11 +265,17 @@ export const editWidgetConfig = (frameId, pageCode) =>
   (dispatch, getState) => {
     const pageConfig = makeGetSelectedPageConfig(pageCode)(getState());
     const pageConfigItem = (pageConfig && pageConfig[frameId]);
-    if (pageConfigItem && pageConfigItem.config) {
-      dispatch(initialize('widgetConfigForm', pageConfigItem.config));
-      history.push(routeConverter(
-        ROUTE_WIDGET_CONFIG,
-        { pageCode, widgetCode: pageConfigItem.code, framePos: frameId },
-      ));
+
+    if (pageConfigItem) {
+      const widget = getWidgetsMap(getState())[pageConfigItem.code];
+      const isConfigurableWidget = (widget && widget.hasConfig) || false;
```

Comment:
I guess that `|| false` isn't really needed

Reply:
Was making sure it's a boolean :) Removed, tests are running locally

Reply:
well, in case of `null` widget it returns null...so we could either keep your version of use something like `(!!widget && widget.hasConfig)`